api/_hyperfunctions/gauge_agg/gauge_zero_time.md

```diff
@@ -0,0 +1,53 @@
+---
+api_name: gauge_zero_time()
+excerpt: Calculate the time when the gauge value is predicted to have been zero
+topics: [hyperfunctions]
+api:
+  license: community
+  type: function
+  toolkit: true
+  experimental: true
+  version:
+    experimental: 1.6.0
+hyperfunction:
+  family: counters and gauges
+  type: accessor
+  aggregates:
+    - gauge_agg()
+api_details:
+  summary: >
+    Calculate the time when the gauge value is predicted to have been zero.
```

Comment:
'predicted' feels like an event that happens in the future, but this is working out something from the past, right? Maybe "calculated"?

Reply:
```suggestion
    Calculate the time when the gauge value is modeled to have been zero.
```